Installer/QModsInstallerScript.iss

```diff
@@ -5,7 +5,7 @@
 #endif
 
 #define Name "QModManager" ; The name of the game will be added after it
-#define Version "3.3.0"
+#define Version "4.0"
```

Comment:
4.0.0

Reply:
Missed that, thanks

Reply:
When I updated to 2.0 there was an issue with ISS thinking that 1.2.1 > 2.0